test/browser/component.js

```diff
@@ -686,3 +686,68 @@ context(`$hooks`, function () {
     });
   });
 });
+
+describe(`Lifecycle Helpers`, function () {
+  let el;
+  let spy;
+
+  beforeEach(function () {
+    document.body.innerHTML = ``;
+    el = document.createElement(`simple-app`);
+    spy = sinon.spy();
+  });
+
+  it(`onConnected doesn't fire until connectedCallback`, async function () {
```

Comment:
you'll want to rejigger these so that "onConnected" and "onDisconnected" are the subjects of `describe` blocks encompassing all of these "it('onConnected..." formulations
```suggestion
  describe(`onConnected`, function() {
    it(`doesn't fire until connectedCallback`, async function () {
```